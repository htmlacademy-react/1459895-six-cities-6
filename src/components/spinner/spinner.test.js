import React from 'react';
import {render} from '@testing-library/react';
import Spinner from './spinner';

describe(`Test Spinner component`, () => {
  it(`Spinner component should render correctly`, () => {
    const {container} = render(
        <Spinner/>
    );
    expect(container).toMatchSnapshot();
  });
});
