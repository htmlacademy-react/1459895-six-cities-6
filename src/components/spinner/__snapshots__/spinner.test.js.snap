// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Spinner component Spinner component should render correctly 1`] = `
<div>
  <div
    class="loadingio-spinner-spin-oi6iug4w1o"
    role="spinner"
  >
    <div
      class="ldio-ta3uggjsjq"
    >
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
    </div>
  </div>
</div>
`;
